apps/modernization-ui/src/apps/patient/profile/delete/DeletePatient.tsx

```diff
@@ -11,23 +11,23 @@ import { DeletabilityResult, resolveDeletability } from './resolveDeletability';
 
 import { useSearchNavigation } from 'apps/search';
 import { Button } from 'components/button';
+import { useProfileContext } from '../ProfileContext';
 
 type Props = {
     patient: Patient;
-    summary: PatientSummary;
 };
 
-const DeletePatient = ({ patient, summary }: Props) => {
+const DeletePatient = ({ patient }: Props) => {
     const { showSuccess, showError } = useAlert();
-
     const { go } = useSearchNavigation();
+    const { summary } = useProfileContext();
```

Comment:
question:  Should the context be used here just to get the summary?  Why not pass the`legalName` into the component?  That will reduce the mocks needed in the unit test....speaking of which

Reply:
Yes Ill add the unit tests, but I was thinking on that context, since `useProfileContext();` is used in the provider of the DeletePatient component in the patient profile component, so if I pass the legalName using the summary from the patient profile to the deletePatient then it will store the previous value when updating the name in demographics not the actual value.